hubblestack/hec/fakelogging.py

```diff
@@ -0,0 +1,28 @@
+
+import os
+import time
+
+class FakeLog(object):
+    ''' teach the hec objects about logging
+        in such a way that it can never accidentally go to slunk
+        (causing an infinite loop of logging)
+    '''
+    def __init__(self, name):
+        self.file = os.environ.get('HUBBLE_HEC_DEBUG')
+        self.name = 'hubblestack.hec.' + name
+    def _show(self, level, fmt, *a):
+        if not self.file:
+            return
+        p = os.getpid()
+        t = time.ctime()
+        with open(self.file, 'a') as fh:
```

Comment:
I don't love avoiding the logging system altogether -- this means that you won't see these emitted on STDERR if running in the foreground, that kind of stuff. We may have to revisit this.